Packs/EmailCommunication/Scripts/SendEmailReply/SendEmailReply_test.py

```diff
@@ -51,6 +52,23 @@ def test_get_email_recipients(email_to, email_from, service_mail, excepted):
     assert result == excepted
 
 
+@pytest.mark.parametrize(
+    "notes, attachments, expected_results",
+    [([{'Metadata': {'user': 'DBot'}, 'Contents': 'note1'}, {'Metadata': {'user': 'DBot'}, 'Contents': 'note2'}],
+     [{'name': 'attachment1.png'}, {'name': 'attachment2.png'}],
+     "Attachments: ['attachment1.png', 'attachment2.png']\n\nDBot: \nnote1\n\nDBot: \nnote2\n\n")
+     ]
+)
+def test_get_reply_body(mocker, notes, attachments, expected_results):
```

Comment:
Please dd `Given When Then`
Also, you have only one case here so using `parametrize` is redundant, let's add a case where no attachments were added as well.